.travis.yml

```diff
@@ -4,94 +4,62 @@
-    - mv $theme_slug $theme_dir
-    # Set up WordPress configuration.
-    - cd $WP_DEVELOP_DIR
-    - echo $WP_DEVELOP_DIR
-    - cp wp-tests-config-sample.php wp-tests-config.php
-    - sed -i "s/youremptytestdbnamehere/wordpress_test/" wp-tests-config.php
-    - sed -i "s/yourusernamehere/root/" wp-tests-config.php
-    - sed -i "s/yourpasswordhere//" wp-tests-config.php
-    # Create WordPress database.
-    #- mysql -u root -e "CREATE DATABASE wordpress_test;"
-    # Install CodeSniffer for WordPress Coding Standards checks.
-    - git clone -b master --depth 1 https://github.com/squizlabs/PHP_CodeSniffer.git $PHPCS_DIR
-    # Install WordPress Coding Standards.
-    - git clone -b master --depth 1 https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards.git $WPCS_DIR
-    # Install PHP Compatibility sniffs.
-    - git clone -b master --depth 1 https://github.com/wimg/PHPCompatibility.git $PHPCOMPAT_DIR
-    # Set install path for PHPCS sniffs.
-    # @link https://github.com/squizlabs/PHP_CodeSniffer/blob/4237c2fc98cc838730b76ee9cee316f99286a2a7/CodeSniffer.php#L1941
-    - $PHPCS_DIR/bin/phpcs --config-set installed_paths $WPCS_DIR,$PHPCOMPAT_DIR
-    # Hop into themes directory.
-    - cd $theme_dir
-    # After CodeSniffer install you should refresh your path.
-    - phpenv rehash
+    # Install composer dependencies
+    - composer install
     # Install JSCS: JavaScript Code Style checker
     # @link http://jscs.info/
-    - npm install -g jscs
+    - if [[ "$PHPCS" == 1 ]]; then npm install -g jscs; fi
     # Install JSHint, a JavaScript Code Quality Tool
     # @link https://jshint.com/docs/
-    - npm install -g jshint
-    - wget https://develop.svn.wordpress.org/trunk/.jshintrc
+    - if [[ "$PHPCS" == 1 ]]; then npm install -g jshint; fi
+    - if [[ "$PHPCS" == 1 ]]; then wget https://develop.svn.wordpress.org/trunk/.jshintrc; fi
 
 # Run test script commands.
 # Default is specific to project language.
 # All commands must exit with code 0 on success. Anything else is considered failure.
 script:
-    # Search theme for PHP syntax errors.
-    - find . \( -name '*.php' \) -exec php -lf {} \;
-    # Run the theme through JSHint
-    - jshint .
-    # Run the theme through JavaScript Code Style checker
-    - jscs .
-    # WordPress Coding Standards
-    # @link https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
-    # @link http://pear.php.net/package/PHP_CodeSniffer/
-    # -p flag: Show progress of the run.
-    # -s flag: Show sniff codes in all reports.
-    # -v flag: Print verbose output.
-    # -n flag: Do not print warnings (shortcut for --warning-severity=0)
-    # --standard: Use WordPress as the standard.
-    # --extensions: Only sniff PHP files.
-    - $PHPCS_DIR/bin/phpcs -p -s -v -n . --extensions=php --ignore=*/woocommerce/*,*/src/*
+  # Validate the composer.json file.
+  # @link https://getcomposer.org/doc/03-cli.md#validate
+  - composer validate --no-check-all --strict
```

Comment:
`composer install` already does a basic validation, so you don't really need to run this command on all PHP versions. Running it on one should be enough.

Reply:
Thank you!